Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```diff
@@ -148,3 +148,29 @@
     price: 75
   - type: Tag
     tags: [ ]
+   
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesBootsJump
+  name: jump boots
+  description: Technological boots used by explorers of new worlds from various unwanted obstacles.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+    layers:
+    - state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+  - type: JumpAbility
+    jumpPower: 3
```

Comment:
Can probably even be increased to 4 or 5. Whe ntesting it felt a little underwhelming.